src/spikeinterface/core/recording_tools.py

```diff
@@ -465,7 +465,7 @@ def write_to_h5_dataset_format(
     return save_path
 
 
-def get_random_recording_slices(
+def get_random_slices(
```

Comment:
This is too neutral no ? Lets find some thing like
get_random_slices_along_time